packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -664,10 +665,11 @@ Or read our docs at https://www.prisma.io/docs/concepts/components/prisma-client
 
       return createPrismaPromise((transaction) => {
         return this._request({
-          args: { command: command },
+          args: command,
           clientMethod: '$runCommandRaw',
           dataPath: [],
           action: 'runCommandRaw',
+          argsMapper: rawCommandArgsMapper,
```

Comment:
Random note: found out in the code that we have one for `rawQueryArgsMapper` but none for runCommandRaw

Nice!